associacao-brasileira-de-normas-tecnicas-ufrgs.csl

```diff
@@ -173,7 +175,7 @@
   </macro>
   <macro name="access">
     <choose>
-      <if type="book chapter">
+      <if type="book chapter" variable="URL" match="all">
```

Comment:
@adam3smith, do you know if this generally works well (since an item can only be of a single type)? Or do we needed a nested conditional here?

Reply:
no, this won't work, you need to test sequentially (the above shouldn't work either, btw., given that mach="all" is the default)